### DISCUSSION TITLE
Added: LDM Notes for April 14th, 2021

### DISCUSSION THREAD

**Discussion by 333fred** [dotnet/csharplang#4663] [LDM Notes]
https://github.com/dotnet/csharplang/blob/main/meetings/2021/LDM-2021-04-14.md

## Agenda

1. Shadowing in record types
2. `field` keyword
3. Improved interpolated strings

**Comment by jnm2** [comment]
I'm so excited about the `field` outcomes!
There's a typo `circularties`.

**Reply by 333fred** [reply]
You don't see anything.

**Reply by alrz** [reply]
I also didn't see the title

**Reply by 333fred** [reply]
Indeed you didn't.

**Reply by bbarry** [reply]
no `change` of being ambiguous

**Reply by 333fred** [reply]
Alright y'all, just submit PRs.
